### PR TITLE
Create Answer plus Edit answer, all test cases passes

### PR DESCRIPTION
**Pull request by AlokSaboji**

### CODE REVIEW

**Comment by anup-shanbhag** [comment]
Looks good!

File: quora-service/src/main/java/com/upgrad/quora/service/business/AnswerBusinessService.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  @Autowired
+  AnswerDao answerDao;
+
+  @Transactional(propagation = Propagation.REQUIRED)
+  public AnswerEntity createAnswer(AnswerEntity answerEntity){
+    return answerDao.createAnswer(answerEntity);
+  }
+
+
```
Comment (ashiksappandi) [inline quora-service/src/main/java/com/upgrad/quora/service/business/AnswerBusinessService.java:22]: Add comments for the methods

File: quora-service/src/main/java/com/upgrad/quora/service/type/AnswerStatus.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.upgrad.quora.service.type;
+
+public class AnswerStatus {
```
Comment (ashiksappandi) [inline quora-service/src/main/java/com/upgrad/quora/service/type/AnswerStatus.java:3]: Complete file is commented

**Comment by AlokSaboji** [comment]
Create Answer plus Edit answer, all test cases passes

**Pull request merged**